editor/src/components/canvas/canvas-strategies/shared-absolute-move-strategy-helpers.ts

```diff
@@ -191,6 +201,7 @@ export function resizeBoundingBox(
   boundingBox: CanvasRectangle,
   drag: CanvasPoint,
   edgePosition: EdgePosition,
+  keepAspectRatio: boolean,
```

Comment:
Please can you change these boolean flags to string unions so that it's clearer which flag is being set

Reply:
done